packages/flutter_tools/lib/src/android/android_workflow.dart

```diff
@@ -15,6 +16,8 @@ import '../globals.dart';
 import 'android_sdk.dart';
 import 'android_studio.dart' as android_studio;
 
+AndroidWorkflow get androidWorkflow => context[AndroidWorkflow];
```

Comment:
This will alleviate us from having to aggressively inject it everywhere (like Fuchsia and google-internal)

```
context.putIfAbsent(AndroidWorkflow, () => new AndroidWorkflow())
```